lib/miq_expression/tag.rb

```diff
@@ -19,11 +20,12 @@ def self.parse(field)
 
   def initialize(model, associations, column, managed = true)
     super(model, associations, column)
-    @namespace = "/#{managed ? MANAGED_NAMESPACE : USER_NAMESPACE}/#{column}"
+    @namespace = managed ? MANAGED_NAMESPACE : USER_NAMESPACE
+    @path = "/#{namespace}/#{column}"
   end
 
   def contains(value)
-    ids = model.find_tagged_with(:any => value, :ns => namespace).pluck(:id)
+    ids = model.find_tagged_with(:any => value, :ns => path).pluck(:id)
```

Comment:
@imtayadeway want to get your thoughts here...I agree with the name change of `namespace`, but there are many references to `namespace` and even just `ns` that should be changed to make all of them align / the name change make sense. I think that is getting out of the scope of this issue. Should we leave them for now and refactor later to make all of the changes in a separate PR, or just do this namespace name change in its entirety in a separate PR?